migrations/003-create-table-blog-posts.ts

```diff
@@ -36,6 +36,20 @@ const blogPosts = [
     is_published: true,
     user_id: 2,
   },
+  {
+    title: "Special Blogpost 2",
+    text_content:
+      'This is a <b>special</b> one <img src="x" onerror=alert(document.cookie) />',
```

Comment:
since we're going to revert the change to the cookies, we need to change the content of this alert here

Reply:
Done in da3dbe4